data.json

```diff
@@ -711,5 +711,17 @@
     "stars": 3888,
     "installCount": 100000,
     "lastUpdate": "30 Jul 2020"
+  },
+  {
+    "name": "GitHub Dark Mode",
+    "source": "https://github.com/MrWillCom/github-dark-mode",
+    "tags": [
+      "dark-mode"
```

Comment:
Values define in the tags fields appeare as filter in the web version. Therefore I would prefer to rename this value to `theme`

Reply:
Thanks, `theme` is really a better choice.

Commt https://github.com/stefanbuck/awesome-browser-extensions-for-github/pull/148/commits/5d83992726ab9ac6ccae4ded9be09f7773a00814 changed this.